external-crates/move/documentation/book/src/control-flow/labeled-control-flow.md

```diff
@@ -0,0 +1,67 @@
+# Labeled Control Flow
+
+Move supports labeled control flow, allowing you to define and transfer control to specific labels
+in a function. For example, we can nest two loops and use `break` and `continue` with those labels
+to precisely specify control flow. Any `loop` or `while` form may be prefixed with a `'label:` form
+to allow us to break or continue directly there.
+
+To demonstrate this behavior, consider a function that takes a nested vector of vectors of numbers
```

Comment:
```suggestion
To demonstrate this behavior, consider a function that takes a nested vector of number vectors
```
Ignore if this changes the meaning slightly. The 'of's aren't really that hard to process.

Reply:
Revised this to hopefully clarify